wolox-books/src/app/screens/sign-up/sign-up.component.scss

```diff
@@ -0,0 +1,75 @@
+@import './src/scss/colors';
+
+.content {
+  background-color: $lightgrey;
+  padding: 15px;
+  border-top: 4px solid $blue;
+  max-width: 350px;
+  width: 100%;
+  margin: auto;
+
+  .header {           
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-bottom: 20px;
+
+    .wolox-logo {
+      width: 100%;
+      max-width: 300px;
+      padding: 0 10px;
+    }
+
+    .title {
+      margin: 5px;
+      text-align: center;
+      letter-spacing: 6px;
+      font-weight: bold;
+    }
+  }   
+  .form {
+    padding-bottom: 20px;
+    border-bottom: 2px solid $alto;
+    margin-bottom: 20px;
+    width: 100%;                
+
+    .text {
+      padding-left: 7px;
+      text-align: left;
+      margin-bottom:4px;
+      font-weight: bold;
+    }
+      
+    .input {
+      width: 100%;
+      border-radius: 5px;
+      padding: 6px 3px;
+      margin-bottom: 12px;
+    }
+  
+
+    .primary-button {
```

Comment:
inputs, primary and secondary buttons will be used in the whole app so it might be a better idea to move them to the generic styles section